networkit/cpp/numerics/test/LAMGGTest.cpp

```diff
@@ -1,121 +1,229 @@
 /*
  * LAMGGTest.cpp
  *
- *  Created on: 20.11.2014
- *      Author: Michael
+ *  Created on: 24.04.2024
```

Comment:
Why are there now two files for testing? I think at least the file names are not self-explanatory in this case.

Reply:
There used to be a `LAMGGtest` which did not actually test the `LAMG` class at all - only `SolverLamg`. I moved these tests to a new test file. I will rename it to `SolverLamgGTest` now.
For new `LAMG` tests I use the now empty `LAMGGTest` file.